env.example

```diff
@@ -0,0 +1,4 @@
+RABBIT_URL=amqp://guest:guest@localhost:5672
```

Comment:
You need to remove this file

Reply:
right. there is no need to use docker-compose either hahaha